spec/features/user_edit.rb

```diff
@@ -0,0 +1,40 @@
+require 'feature_helper'
+
+RSpec.feature 'Edit User' do
+  context 'I am a member I am logged in' do
+    let(:member) { create(:user) }
+    before(:each) { create_current_user(member) }
+
+    context 'I can edit my own information' do
+      before { visit edit_user_path(member) }
```

Comment:
Should probably get rid of this before block and just move the visit into the scenario

      scenario 'should be able to edit my info using appropriate data' do
        visit edit_user_path(member)
        fill_in 'Name', with: 'another name'
        click_button('Submit')
        expect(page).to have_content('Successfully Updated')
      end